components/BottomNavigation/tests/unit/MDCBottomNavigationBarControllerTests.m

```diff
@@ -338,9 +442,9 @@ - (BOOL)bottomNavigationBarController:
   return [self.delegateReturnValue boolValue];
 }
 
-#pragma mark - Helper Methods
+#pragma mark Helper Methods
```

Comment:
Nit: I would prefer to keep the divider.  In either case, please separate the refactoring/clean-up to a different PR.

Reply:
Done.